### PR TITLE
Extend timestamp with date

### PR DESCRIPTION
**Pull request by ankcn**
Print current date in timestamp when -Q option is used.
Extra new line added after each report.
Example output:
[05:47:24 17.12.2021]
ya.ru      : xmt/rcv/%loss = 30/30/0%, min/avg/max = 47.1/47.2/47.2
he.net     : xmt/rcv/%loss = 30/30/0%, min/avg/max = 222/222/223
google.com : xmt/rcv/%loss = 30/30/0%, min/avg/max = 55.0/55.1/55.2

[05:47:54 17.12.2021]
ya.ru      : xmt/rcv/%loss = 30/30/0%, min/avg/max = 47.1/47.2/47.2
he.net     : xmt/rcv/%loss = 30/30/0%, min/avg/max = 222/222/223
google.com : xmt/rcv/%loss = 30/30/0%, min/avg/max = 55.0/55.1/55.2

### CODE REVIEW

**Comment by schweikert** [comment]
This breaks compatibility so it can't be merged.
